src/mmw/js/src/core/opacityControl.js

```diff
@@ -0,0 +1,80 @@
+/*
```

Comment:
This should probably go in https://github.com/WikiWatershed/model-my-watershed/tree/develop/src/mmw/js/shim